blocks/wizard/wizard.js

```diff
@@ -177,18 +177,7 @@ export default async function decorate(block) {
           <a href="/templates/${id}" id="${id}" class="template ${i === 0 ? 'is-selected' : ''}">
             <h3>${name}</h3>
             <p>${description}</p>
-            <div class="carousel">
```

Comment:
Why did you remove the carousel ? Is it because you're not supposed to allow other clickable elements inside a `<a>` ?

Reply:
We decided to replace the carousel with a static image.
It was a little bit hard to change between the images.
A Single Image for a quick preview should be okay, as Users can preview the Site.